src/main/io/motors.h

```diff
@@ -24,6 +24,7 @@ typedef struct motorConfig_s {
     uint16_t maxthrottle;                   // This is the maximum value for the ESCs at full power this value can be increased up to 2000
     uint16_t mincommand;                    // This is the value for the ESCs when they are not armed. In some cases, this value must be lowered down to 900 for some specific ESCs
     uint16_t motor_pwm_rate;                // The update rate of motor outputs (50-498Hz)
```

Comment:
`uint16_r reserved1;` between `mincommand` and `motor_pwm_rate` to take place of former `servoCenterPulse`. 
That way all motor fields stay valid after this change and there is no need to bump record version (and values are not reset)